Modules/xASL_module_ASL.m

```diff
@@ -415,6 +415,8 @@
 		x.P.Path_PWI4D_used = x.P.Path_PWI4D;
 		x.P.Pop_Path_PWI4D_used = x.P.Pop_Path_PWI4D;
 	end
+	% Note: The paths Path_PWI4D_used can differ from a simple subtraction as it can point out to version that has some outlying volumes excluded.
```

Comment:
Change this note to:

% Note: The paths x.P.(Pop_)Path_PWI4D_used can differ from a PWI4D obtained from a simple subtraction as we may have merged sessions, and the user or ExploreASL may have removed volumes from PWI4D

Reply:
Agree. Done.